tests/rptest/scale_tests/audit_log_test.py

```diff
@@ -393,24 +393,26 @@ def test_audit_log(self):
         self._enable_auditing()
         audit_enabled_results = self._run_repeater(topic_names, scale)
 
-        # Assert that there is no more then a x% difference in produce and consume throughput
-        allowable_threshold = 10.0
-
         def pct_difference(a, b):
             return (abs(a - b) / abs((a + b) / 2)) * 100
```

Comment:
Sorry not part of the diff, but looking at this again it's not clear what's being measured. Intuitively it seems like if `a=10` and `b=8`, then throughput got 20% worse compared to baseline, but this gives ~22.2%.

```suggestion
            return abs(b - a) / a * 100
```

is what I would have expected, but it's entirely possible that this is meant to express something different.

Reply:
I think theres two formulas, one called `percent difference` another `percent change` (which you mentioned) https://www.calculatorsoup.com/calculators/algebra/percent-difference-calculator.php

Reply:
I think percent change would be more applicable +1

Reply:
Oh, ya fair enough. Whatever seems most useful to you in context 👍